template/_templates/component/new/inject_story.ejs

```diff
@@ -1,8 +1,8 @@
 ---
 inject: true
 to: storybook/stories/index.ts
-skip_if: components/<%= name %>
+skip_if: components/<%= h.inflection.transform(name, ['undasherize', 'classify']) %>
 append: true
 ---
-<% formattedName = name -%>
-import '../../src/components/<%= formattedName %>/stories'
\ No newline at end of file
+<% formattedName = h.inflection.transform(name, ['undasherize', 'classify']) -%>
+import '../../src/components/<%= formattedName %>/<%= formattedName %>.stories'
```

Comment:
Great update!! Should this have the semi-colon at the end to stay consistent with linter?

Reply:
✨